packages/orbit-components/src/Inline/__tests__/index.test.js

```diff
@@ -0,0 +1,99 @@
+// @flow
+import React from "react";
+import { screen, render } from "@testing-library/react";
+
+import theme from "../../defaultTheme";
+import Inline from "..";
+
+const dataTest = "test";
+
+const tokens = {
+  none: "",
+  "xx-small": theme.orbit.spaceXXSmall,
+  "x-small": theme.orbit.spaceXSmall,
+  small: theme.orbit.spaceSmall,
+  medium: theme.orbit.spaceMedium,
+  large: theme.orbit.spaceLarge,
+  "x-large": theme.orbit.spaceXLarge,
+  "xx-large": theme.orbit.spaceXXLarge,
+  "xxx-large": theme.orbit.spaceXXXLarge,
+};
+
+const Elements = () => (
+  <>
+    <div data-test="child-0">kek</div>
+    <div>bur</div>
+    <div>cheburek</div>
+    <div>blin</div>
+  </>
+);
+
+describe("#Inline", () => {
+  it("should have props", () => {
+    render(
+      <Inline align="center" justify="start" dataTest={dataTest}>
+        <Elements />
+      </Inline>,
+    );
+    expect(screen.getByTestId(dataTest)).toBeInTheDocument();
+    const styles = getComputedStyle(screen.getByTestId(dataTest));
+    expect(styles).toHaveProperty("justify-content", "center");
+    expect(styles).toHaveProperty("align-items", "start");
+  });
+
+  it("should have spacings", () => {
+    const spacings = Object.keys(tokens).map(token => (
+      <Inline key={token} align="start" justify="center" spacing={token} dataTest={dataTest}>
+        <div data-test={token} style={{ marginLeft: tokens[token] }}>
+          kek
+        </div>
+        <div>bur</div>
+      </Inline>
+    ));
+
+    expect(getComputedStyle(render(spacings[0]).getByTestId("none"))).toHaveProperty(
+      "margin-left",
+      "",
+    );
```

Comment:
I recommend deleting this `it` block entirely, you're just asserting that `spacing` maps to correct tokens, and a mistake happening there is just as likely as making a mistake in the test file itself. 😉